assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -56,6 +56,7 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi, encodeMarkup }) => {
       sortOption = STORY_SORT_OPTIONS.LAST_MODIFIED,
       sortDirection,
       searchTerm,
+      author,
```

Comment:
😕

Reply:
This is change was introduce because `Selected Stories` block uses some of the existing components. We have added authors' dropdown in the story picker modal. Shall we remove these modifications from the existing component?

Reply:
This change has happened in the dashboard not your block. That is why the confused face.

Reply:
This change is done to enable filtering stories by the author in the story picker modal. Do you suggest to write maybe a custom APIProvider for this block. then this change won't be needed.

Reply:
Oh you are using the dashboard's api provider. If so, then yes, this code should have it's own api provider and not piggyback on top of the dashboard's one. I dont want to break in dashboard to break this block as well. CC @swissspidy

Reply:
also, apart from this, the current implementation uses some of the dashboard's components. Will that be okay or shall we completely decouple the block from the dashboard.

Reply:
Using the components should be OK, but I would not directly use the providers.